rclcpp_action/CMakeLists.txt

```diff
@@ -98,6 +98,16 @@ if(BUILD_TESTING)
       ${PROJECT_NAME}
     )
   endif()
+
```

Comment:
this PR also adds a lot of tests for `rclcpp_action`, were those added intentionally here?
Seems that the same tests were added in https://github.com/ros2/rclcpp/pull/1043